src/main/java/io/smartcat/migration/Migration.java

```diff
@@ -45,18 +47,70 @@ public int getVersion() {
 
     /**
      * Returns migration description (for history purposes)
-     * 
-     * @return Migration description
      */
     public abstract String getDescription();
 
 	/**
      * Executes migration implementation
-     * 
-     * @throws MigrationException
+     *
+     * @throws MigrationException exception
      */
     public abstract void execute() throws MigrationException;
 
+    /**
+     * Execute provided statement and checks if the schema migration has been propagated
+     * to all nodes in the cluster. Use this method when executing schema migrations.
+     * @param statement Statement to be executed
+     * @throws MigrationException exception
+     */
+    protected void executeWithSchemaAgreement(Statement statement)
+            throws MigrationException
+    {
+        ResultSet result = this.session.execute(statement);
+        if (checkSchemaAgreement(result)) return;
```

Comment:
Let's not write one line if statements, they are not readable (I know that this is personal opinion but almost all projects in Java are using new lines and code block so at least all developers would be more familiar to that code style.